pkg/lightning/backend/tidb/tidb.go

```diff
@@ -365,30 +381,80 @@ func (be *tidbBackend) ImportEngine(context.Context, uuid.UUID) error {
 
 func (be *tidbBackend) WriteRows(ctx context.Context, _ uuid.UUID, tableName string, columnNames []string, rows kv.Rows) error {
 	var err error
-outside:
+rowLoop:
 	for _, r := range rows.SplitIntoChunks(be.MaxChunkSize()) {
+		batch := true
+	retryLoop:
 		for i := 0; i < writeRowsMaxRetryTimes; i++ {
-			err = be.WriteRowsToDB(ctx, tableName, columnNames, r)
+			err = be.WriteRowsToDB(ctx, tableName, columnNames, r, batch)
 			switch {
 			case err == nil:
-				continue outside
+				continue rowLoop
```

Comment:
Why not use `break`

Reply:
Using `break` here will trigger the line 403.